bluepysnap/edges.py

```diff
@@ -903,6 +846,10 @@ def iter_connections(
             return map(omit_edge_count, it)
 
     @property
-    def h5_filepath(self):
+    def h5_filepath(self):  # pylint: disable=inconsistent-return-statements
         """Get the H5 edges file associated with population."""
-        return self._edge_storage.h5_filepath
+        for edge_conf in self.circuit.config["networks"]["edges"]:
+            h5_filepath = edge_conf["edges_file"]
+            # set(...) as pylint seems to think {}.population_names doesn't return a set
```

Comment:
If pylint is wrong here, disabling the error would be better, no?

Reply:
Yeah, fixed